git_hooks/post-checkout

```diff
@@ -2,4 +2,4 @@
 #
 # Writes git revision into git_revision file
 #
-git show-ref refs/heads/master | cut -d " " -f 1  > git_revision
+git rev-parse HEAD | cut -d " " -f 1  > git_revision
```

Comment:
>  I am not sure why you didn't use --short in the first place

Because I just found one way to get the desired result and didn't care about the best way to do it.

> Travis build failed btw and it's quite relevant... I'm unfamiliar with travis so far (only used mocha)... but learning as I go along here.

No, just issue with Qt 5.7 PPA.